src/test-utils/pages-and-contexts.ts

```diff
@@ -0,0 +1,82 @@
+export const pagesAndContexts: Record<string, (string | undefined)[]> = {
+  "check-mobile-app-result": [],
+  "confirm-your-details": [],
+  "delete-handover": [],
+  "find-another-way-access-service": [],
+  "live-in-uk": [],
+  "no-photo-id-abandon-find-another-way": [],
+  "no-photo-id-exit-find-another-way": [],
+  "no-photo-id-security-questions-find-another-way": ["dropout", undefined],
+  "non-uk-no-app-options": [],
+  "non-uk-app-intro": [],
+  "non-uk-no-app": [],
+  "non-uk-no-passport": [],
+  "non-uk-passport": [],
+  "page-dcmaw-success": ["coiNoAddress"],
```

Comment:
Not related to your change (so feel free to do separately), but should this have `undefined` as well?

Reply:
nice spot!